src/main/java/com/marklogic/contentpump/Command.java

```diff
@@ -1239,6 +1317,20 @@ public void configOptions(Options options) {
                             "Input Truststore password to use for SSL connections")
                     .create(INPUT_TRUSTSTORE_PASSWD);
             options.addOption(inputTruststorePasswd);
+            Option inputBasePath = OptionBuilder
+                .withArgName("string")
```

Comment:
is "string" a good argName? Isn't it normally the option/parameter name or something descriptive?

Reply:
That's a good point Mads...I think this is a general issue for MLCP options.